modules/viewing-accelerator-profiles.adoc

```diff
@@ -0,0 +1,25 @@
+:_module-type: PROCEDURE
+
+[id='viewing-accelerator-profiles_{context}']
+= Viewing accelerator profiles
+If you have defined accelerator profiles for {productname-short}, you can view them from the *Accelerator profiles* page. You can also  enable or disable an accelerator profile from this page. 
```

Comment:
Could remove the last sentence and just update to 

'If you have defined accelerator profiles for {productname-short}, you can view, enable, and disable them from the *Accelerator profiles* page.'

or, alternatively

'If you have defined accelerator profiles for {productname-short}, you can manage them from the *Accelerator profiles* page.'